packages/gatsby-theme-newrelic/gatsby/type-defs.graphql

```diff
@@ -52,3 +53,13 @@ type NewRelicThemeTessenConfig {
   product: String
   subproduct: String
 }
+
+type googleTagManagerConfig {
+  trackingId: String
+  src: String
+  options: googleTagManagerOptions
+}
+
+type googleTagManagerOptions {
+  anonymize_ip: Boolean
```

Comment:
Out of curiosity, why is this snake_case vs camelCase?

Reply:
oh i just mirrored the syntax expected in the `gtag()` options arg.